documents/package.json

```diff
@@ -15,7 +15,8 @@
     "author": "Project Jupyter Contributors",
     "files": [
         "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
-        "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+        "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
+        "src/**/*.{ts,tsx}"
```

Comment:
Because this adds a new line you need to adjust line numbering in `README.md`:

https://github.com/jupyterlab/extension-examples/blob/4585387bd2183b02bd4c9782d716e006d8cb1cbc/documents/README.md?plain=1#L38-L47

Currently CI fails with:

```
   documents/README.md#L38-L47 Embedded 6 lines from file package.json#L108-L113
```